Bottom: 604d32b55a1837315eec2ea84d87d05e08481ecc
Top:    99fc79f402f5fd121fbe4369ee97a7a596a55b77
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-07-12 11:12:22 -0500

Install buildbot plugin dependencies


---
diff --git a/docker/buildbot-master/Dockerfile b/docker/buildbot-master/Dockerfile
index 0947a60c..1baa3f49 100644
--- a/docker/buildbot-master/Dockerfile
+++ b/docker/buildbot-master/Dockerfile
@@ -14,7 +14,11 @@ RUN apk --no-cache add \
 
 # install pip dependencies
 RUN pip --no-cache-dir install \
-    'attrs'
+    'attrs' \
+    'buildbot-console-view' \
+    'buildbot-grid-view' \
+    'buildbot-waterfall-view' \
+    'buildbot-www'
 
 # Create buildbot user
 ARG BUILDBOT_UID=1000
